prow/plugins/project/project.go

```diff
@@ -284,12 +294,25 @@ func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent, p
 
 	// Move this issue/PR to the new column if there's already a project card for
 	// this issue/PR in this project
-	existingProjectCard, err := gc.GetColumnProjectCard(proposedColumnID, e.Number)
-	if err != nil {
-		return err
+	var existingProjectCard *github.ProjectCard
+	for _, colID := range projectColumns {
+		existingProjectCard, err = gc.GetColumnProjectCard(colID.ID, e.IssueHTMLURL)
```

Comment:
Why do we overwrite `existingProjectCard` on every loop?

Reply:
Oh nevermind -- maybe add a `break` here and put the column check after we finish the loop for clarity?

Reply:
Done, I needed to bring the column ID out.